unlocks.csv

```diff
@@ -300,9 +299,11 @@ Analysis|BaseUpgradeConsole|15,1,BaseUpgradeConsole
 Analysis|Workbench|0,1,Workbench
 Analysis|Cyclops|0,1,Cyclops
 Analysis|Cyclops|1,1,Cyclops
+Analysis|Exosuit|0,1,Exosuit
 Analysis|HydrochloricAcid|0,1,HydrochloricAcid
 Analysis|JellyPlant|0,1,
 Analysis|Lubricant|0,0,Lubricant
+Analysis|FiberMesh|0,0,FiberMesh|Knife
```

Comment:
Flagged as non-standard, not sure this is normally an unlock? You start with first aid kit, which is the only thing I could think of (synthetic fibres is from blood oil alongside benzene)